### PR TITLE
Issues/refactor models

### PR DESCRIPTION
**Pull request by mkoistinen**
Addresses #59.

### CODE REVIEW

File: aldryn_jobs/models.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ class JobCategory(TranslatableModel):
 
     supervisors = models.ManyToManyField(
         get_user_model_for_fields(), verbose_name=_('Supervisors'),
-        related_name='job_offer_categories',
+        related_name='job_offer_categories',  # FIXME: This is mis-named should be "job_categories"?
```
Comment (czpython) [inline aldryn_jobs/models.py]: considering the model name is job offer, I don't think this is misnamed.

File: aldryn_jobs/models.py [hunk 2]
```diff
@@ -241,10 +239,7 @@ def get_absolute_url(self, language=None):
         category_slug = self.category.safe_translation_getter(
             'slug', language_code=language
         )
-        if self.app_config_id:
-            namespace = self.app_config.namespace
-        else:
-            namespace = 'aldryn_jobs'
+        namespace = getattr(self.category.app_config, "namespace", "aldryn_jobs")
```
Comment (czpython) [inline aldryn_jobs/models.py]: under what circumstance would this throw an ```AttributeError```? would there be a case where category does not have app_config?

File: aldryn_jobs/models.py (hunk 1 above)
Comment (mkoistinen) [inline aldryn_jobs/models.py]: But the model is JobCategory, not JobOffer.

File: aldryn_jobs/models.py (hunk 2 above)
Comment (mkoistinen) [inline aldryn_jobs/models.py]: Perhaps, better safe than sorry.

Comment (czpython) [inline aldryn_jobs/models.py]: the reason I bring it up is, if you do this here then you'd have to do it in other places where we rely on app_config.

**Comment by czpython** [comment]
LGTM

**Pull request merged**